File: src/backend/studio-reservation/src/main/java/org/taonaw/studio_reservation/usecase/command/staff/registerStaffAccount/RegisterStaffAccountCommand.java
package org.taonaw.studio_reservation.usecase.command.staff.registerStaffAccount;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Getter;
import org.taonaw.studio_reservation.domain.model.shared.PersonName;
import org.taonaw.studio_reservation.domain.model.shared.PlainTextPassword;
import org.taonaw.studio_reservation.domain.model.staffAccount.LoginId;

@Getter
@Builder
@AllArgsConstructor
public class RegisterStaffAccountCommand {
    private PersonName name;
    private LoginId loginId;
    private PlainTextPassword plainTextPassword;
}

File: src/backend/identityaccess/src/main/kotlin/org/taonaw/identityaccess/usecase/staffAccount/registerStaffAccount/RegisterStaffAccountCommand.kt
package org.taonaw.identityaccess.usecase.staffAccount.registerStaffAccount

import org.taonaw.identityaccess.domain.model.staffAccount.StaffName
import org.taonaw.identityaccess.domain.model.staffAccount.UserId
import org.taonaw.identityaccess.domain.model.shared.PlainTextPassword

data class RegisterStaffAccountCommand(
        val staffName: StaffName,
        val userId: UserId,
        val plainTextPassword: PlainTextPassword) {
}